### PR TITLE
Add blurring API

### PR DESCRIPTION
**Pull request by raikasdev**
Adds the qrpyora-blurring to backend

### CODE REVIEW

**Review by Chicken — CHANGES_REQUESTED** [review]
The docker image `ghcr.io/testausserveri/qrpyora-blur/qrpyora-blur` isn't built for `linux/arm64` and therefore won't run on our servers.

**Comment by ahnl** [comment]
Does the backend save both an original untouched and a blurred version?

**Review by Chicken — APPROVED** [review]

File: backend/package.json [hunk 1]
```diff
@@ -13,13 +13,13 @@
     "express": "^4.17.1",
```
Comment (developerfromjokela) [inline backend/package.json:13]: Just revert changes? What exactly has changed in this file besides different ordering?

**Review by developerfromjokela — CHANGES_REQUESTED** [review]

File: backend/package.json (hunk 1 above)
Comment (ahnl) [reply backend/package.json:13]: not a biggie, I'd pass it

**Review by developerfromjokela — APPROVED** [review]

**Comment by ahnl** [comment]
Hilarious, I like it.

image

**Review by ahnl — APPROVED** [review]

**Pull request merged**